docs/usage/getting-started/running.md

```diff
@@ -173,16 +192,16 @@ Read the platform-specific docs to learn how to setup authentication on your pla
 
 ### GitHub.com token for release notes
 
-If you are running on any platform except github.com, it's important to also configure the environment variable `GITHUB_COM_TOKEN` containing a Personal Access Token for github.com.
-This account can actually be _any_ account on GitHub, and needs only read-only access.
+If you are running on any platform except github.com, you should also set the environment variable `GITHUB_COM_TOKEN` and put the Personal Access Token for github.com in it. QUESTION: this should probably be encrypted somehow???
```

Comment:
```suggestion
If you are running on any platform except github.com, you should also set the environment variable `GITHUB_COM_TOKEN` and put the Personal Access Token for github.com in it.
```

No, it's common - and in some cases best - practice to put secrets in environment variables